README.md

```diff
@@ -78,7 +78,7 @@ using configuration options:
  * `-Darm_neon=false` - will disable the ARM NEON fast paths
  * `-Dgcc_vector=false` - will disable the GCC vector intrinsics
 
-If you don't plan on generating introspection data, use `-Dintrospection=false`
+If you don't plan on generating introspection data, use `-Dintrospection=disabled`
 when configuring Graphene; similarly, if you don't plan on using GObject with
 Graphene, use `-Dgobject_types=false`. Disabling GObject types will also
```

Comment:
Maybe `gobject_types` should also be a feature option?

Reply:
Maybe, but it's unrelated.